docs/source/txflow.rst

```diff
@@ -122,7 +122,7 @@ transaction was validated or invalidated.
           whether your transaction has actually been ordered, validated, and
           committed to the ledger.
 
-See the :ref:`sequence diagram <swimlane>` to better understand the
+See the `sequence diagram <images/flow-4.png>`_ to better understand the
```

Comment:
This link still did not work. RST image links may be a bit different than markdown. Try the more classic format with a short description:
```
.. image:: images/flow-4.png
```

Reply:
HI @obermeier, are you planning on addressing the comment above? The link is still broken with this change. If not I can close this PR and fix the image.